Runtime/Scripts/SelectionTools/PolygonInput.cs

```diff
@@ -72,14 +72,16 @@ public enum WindingOrder
 
         [SerializeField] private LineRenderer polygonLineRenderer;
         [SerializeField] private LineRenderer previewLineRenderer;
-        private List<Vector3> positions = new List<Vector3>();
-        private Vector3 lastAddedPoint = default;
-        private Vector3 selectionStartPosition = default;
-        private Vector3 currentWorldCoordinate = default;
-        private Vector3 previousFrameWorldCoordinate = default;
-        private Vector2 previousFrameScreenCoordinate = default;
-        private Vector3 lastNormal = Vector3.zero;
-        private Plane worldPlane;
+
+        public Vector3 currentWorldCoordinate = default;
+        public List<Vector3> positions = new List<Vector3>();
+
+        protected Vector3 lastAddedPoint = default;
```

Comment:
Ik zou in een toekomstige refactor kijken hoe we ook deze protected fields weer private kunnen maken. Ik neem aan dat je deze open gegooid hebt omdat je deze wil kunnen wijzigen, maar op deze manier is er geen controle meer over het hoe en waarom. 

Vaak is het dan beter om een extra functie op te nemen in deze class die doet wat je wil en als extensionpoint gebruikt kan worden; dan is er namelijk controle over de internal state van dit object. Nu is de controle er niet meer en kan je in methodes in deze class geen aannames meer maken over de correctheid van deze fields (omdat deze in een child class aangepast kan worden en je niet weet wat voor logica er in de child class zit).

Maar: ik ga deze evenzogoed approven omdat ik denk dat het een mooie volgende stap kan zijn en niet perse voor nu is